lib/rules/jsx-closing-bracket-atomic.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * @fileoverview Ensures the "/>" and "</" closing indicators are written as a single token
+ * @author Diogo Franco (Kovensky)
+ */
+'use strict';
+
+// ------------------------------------------------------------------------------
+// Constants
+// ------------------------------------------------------------------------------
+
+var MESSAGE = 'JSX tag closing has embedded whitespace';
```

Comment:
Hm... conceptually, at least in my head, a rule that checks for `</ foo>` should also check for `< foo>` and `< foo/>`.